common/logger/tests/derive.rs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use libra_log_derive::Schema;
```

Comment:
Might it make more sense to make these cargo doc / examples with some text around them on what they're doing?

Reply:
Good point, I'll add some docs to the tests